app/schemas.py

```
@@ -94,3 +94,151 @@
     "UserUpdate",
 ]
+from datetime import datetime, time
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, Field
+
+from app.ai.voice.agents.automatic.types.models import TTSProvider, VoiceName
+
+
+class OutboundNumberStatus(str, Enum):
```

Ruff (0.14.10)

Redefinition of unused `OutboundNumberStatus` from line 55

(F811)